common/generated_libs/network_protocol/network_protocol.pb.h

```diff
@@ -32,13 +32,13 @@ typedef struct _Alarm {
     AlarmKind kind;
 } Alarm;
 
-typedef struct _SensorReadings {
+typedef struct _SensorsProto {
     float patient_pressure_cm_h2o;
     float volume_ml;
     float flow_ml_per_min;
     float inflow_pressure_diff_cm_h2o;
     float outflow_pressure_diff_cm_h2o;
```

Comment:
Will we still need those two pressure_diff to be in the SensorsProto struct? I have a feeling these were here for debugging reasons, and if we have a more practical SensorsReadings, it makes sense not to keep those here

Reply:
You're right, we can probably get rid of them now that we have the debugging API Steve wrote.

Let me send a separate PR for that.